### ISSUE TITLE
critical points of this program

### ISSUE THREAD

**Issue by cryptoprogramer** [Houzich/CUDA-Brute-Force-Mnemonic-Legacy-SegWit#7]
first. what is the point of writing this program in cuda, while there is no speed boost?
python program is twice faster that this cuda. a cuda program for seed must have at least 5 million seed generation per second which i have.
second: the program generate random seed which will lead to no where. it needs to ask user to choose some words and generte the rest randomly.

**Comment by Houzich** [comment]
This program is not for generating mnemonics, but for brute force. You are not here. Learn some theory.

**Comment by cryptoprogramer** [comment]
yes it bruteforces by generating seed, but its bruteforcing speed is very very low bro
in the  instruction you have mentioned that it generates mnemonic:
as translated to English:
number_of_generated_mnemonics": 18000000000000000000 - The total number of mnemonics we want to generate. This was introduced to check the speed of generation or to save the results of generation to files. If we want to be infinite, then set the maximum value to 18000000000000000000.

**Comment by Houzich** [comment]
I do not understand what you're saying. If you have a faster program then don't use this code
